bindings/python/Cargo.toml

```diff
@@ -14,7 +14,7 @@ serde = { version = "1.0", features = ["rc", "derive"] }
 serde_json = "1.0"
 libc = "0.2"
 env_logger = "0.11"
-pyo3 = { version = "0.23", features = ["abi3", "abi3-py39"] }
+pyo3 = { version = "0.23", features = ["abi3", "abi3-py39", "py-clone"] }
```

Comment:
for this we probably need to rebase!

Reply:
Don't think so, @Narsil removed it in https://github.com/huggingface/tokenizers/commit/f4c9fd7f402fc794df8f1b547a95ee5305f9fe62